app/models/teammate.rb

```diff
@@ -45,6 +45,11 @@ def name
     user ? user.name : ""
   end
 
+  def ratings_count(current_event)
+    all_ratings = self.user.ratings.for_event(current_event)
+    all_ratings.select { |r| r.not_withdrawn }.count
```

Comment:
Ohh I see where that's being used. Hmm I'd like to see if the filtering on proposal status can be done in the db instead of in the app.